prime-dotnet-webapi/Models/EnrolleeLinkedEnrolment.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Newtonsoft.Json;
+
+namespace Prime.Models
+{
+    [Table("EnrolleeLinkedEnrolment")]
+    public class EnrolleeLinkedEnrolment : BaseAuditable
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [JsonIgnore]
+        public Enrollee Enrollee { get; set; }
+
+        public int EnrolleeId { get; set; }
```

Comment:
Another super minor point: my convention is to put the Navigation properties right under the foreign key properties rather than the other way around. so 
```c#
public int EnrolleeId { get; set; }
 
[JsonIgnore]
public Enrollee Enrollee { get; set; }

public int? PaperEnrolleeId { get; set; }

[JsonIgnore]
public Enrollee PaperEnrollee { get; set; }
```

Reply:
Moved